### ISSUE TITLE
French/Finnish/Spanish/etc are labelled (en)

### ISSUE THREAD

**Issue by YairRand** [Sefaria/Sefaria-Project#272]
Text version names for non-English languages that use the Latin alphabet are labelled with "(en)". For certain cases, this is also after the language's own ISO code in square brackets ("[fi] (en)"), which looks very wrong, and the meaning of the (en) is potentially confusing to the reader.

**Comment by blockspeiser** [comment]
Thanks for the report. The way this is right now is a stopgap until we get
to properly supporting non-english translations in the backend. We got a
good number of requests from people who wanted to add non-enlgish
translations, so we OK'd them with these guidelines:
https://github.com/Sefaria/Sefaria-Project/wiki/Adding-Non-English-Translations.
I agree it's not as clear as it could be, but it was a compromise to allow
some place for these translation to go.

**Comment by pzp1997** [comment]
How about if we (at least temporarily) just change how the version language is displayed on the front-end in these cases? It does not seem too hard to parse the version string to look for square brackets and replace the language code with that text in s2.js.

**Comment by blockspeiser** [comment]
We're now labeling things correctly on the frontend, even though in the DB they can still be technically marked as English.
